pages/home/crewai/_meta.ts

```diff
@@ -1,4 +1,4 @@
 export default {
   "use-arcade-tools": "Using Arcade tools",
-  "custom-auth-flow": "Custom auth flow",
+  "custom-auth-flow": "Custom Auth Flow",
```

Comment:
```suggestion
  "custom-auth-flow": "Custom auth flow",
```

AFAIK all of our other TOC items are sentence case.

Reply:
good point. updated